Source/Elasticurl/Elasticurl.swift

```diff
@@ -272,30 +267,22 @@ struct Elasticurl {
             var stream: HttpStream?
             let path = context.url.path == "" ? "/" : context.url.path
             let httpRequest: HttpRequest = try HttpRequest(method: context.verb, path: path, allocator: allocator)
-            let headers = try HttpHeaders(allocator: allocator)
-            if headers.add(name: "Host", value: host),
-               headers.add(name: "User-Agent", value: "Elasticurl"),
-               headers.add(name: "Accept", value: "*/*"),
-               headers.add(name: "Swift", value: "Version 5.4") {
-                for header in context.headers {
-                    _ = headers.add(name: header.key, value: header.value)
-                }
-            }
+            var headers = [HttpHeader]()
+            headers.append(HttpHeader(name: "Host", value: host))
```

Comment:
trivial: seems like an alternate constructor that accepted `name` and `value` as unnamed args would be nice

Reply:
Thanks, I have added a addHeader function that takes name & value and removed compression.